src/metabase/models/card.clj

```diff
@@ -81,6 +81,29 @@
      {:Metric  (extract-ids :metric inner-query)
       :Segment (extract-ids :segment inner-query)})))
 
+(defn card->multi-cards
+  "Return the cards which are other cards with respect to this card
+  in multiple series display for dashboard, for a specified dashboard-card.
+
+  Dashboard (and dashboard only) has this thing where you're displaying multiple cards entirely.
+
+  This is actually completely different from the combo display,
+  which is a visualization type in visualization option.
+
+  This is also actually completely different from having multiple series display
+  from the visualization with same type (line bar or whatever),
+  which is a separate option in line area or bar visualization"
+  [{:keys [id]} dashcard]
```

Comment:
why does this take a card? The first arg is unused. Is it just to tie it to the card namespace?

Reply:
because i just whacked it from the hydration function, lol

willfix